index.html

```diff
@@ -0,0 +1,130 @@
+          <li><a href="#declare">Declaring variable</a></li>
+          <li><a href="#scope">Variable scope</a></li>
+          <li><a href="#global">Global variables</a></li>
+          <li><a href="#constant">Constants</a></li>
+          <li><a href="#data">Data types</a></li>
+          <li><a href="#condition">if...else statement</a></li>
+          <li><a href="#function">Function declarations</a></li>
+          <li><a href="#refrence">Refrence</a></li>
+        </ul>
+      </div>
+      <!-- end of side navigation -->
+      <div class="main">
+        <div class="intro">
+          <h2>Introduction</h2>
+          <p>
+            JavaScript is a cross-platform, object-oriented scripting language.
+            It is a small and lightweight language. Inside a host environment
+            (for example, a web browser), JavaScript can be connected to the
+            objects of its environment to provide programmatic control over
+            them.
+          </p>
+          <p>
+            JavaScript contains a standard library of objects, such as Array,
+            Date, and Math, and a core set of language elements such as
+            operators, control structures, and statements. Core JavaScript can
+            be extended for a variety of purposes by supplementing it with
+            additional objects; for example:
+          </p>
+          <ul>
+            <li>
+              Client-side JavaScript extends the core language by supplying
+              objects to control a browser and its Document Object Model (DOM).
+              For example, client-side extensions allow an application to place
+              elements on an HTML form and respond to user events such as mouse
+              clicks, form input, and page navigation.
+            </li>
+            <li>
+              Server-side JavaScript extends the core language by supplying
+              objects relevant to running JavaScript on a server. For example,
+              server-side extensions allow an application to communicate with a
+              database, provide continuity of information from one invocation to
+              another of the application, or perform file manipulations on a
+              server.
+            </li>
+          </ul>
+        </div>
+        <!-- end of intro -->
+        <div class="what">
+          <h2>What you should already know</h2>
+          <p>This guide assumes you have the following basic background:</p>
+          <ul>
+            <li>
+              A general understanding of the Internet and the World Wide Web
+              (WWW).
+            </li>
+            <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+            <li>
+              Some programming experience. If you are new to programming, try
+              one of the tutorials linked on the main page about JavaScript.
+            </li>
+          </ul>
+        </div>
+        <!-- end of what -->
+        <div class="js">
```

Comment:
Please use descriptive class names. Using standardized class names and following the naming convention will not only save your work but also when you'll work in a company in the future it'll feel like it was written by the same person. There are different naming conventions for CSS class names. 

Please read this guide of [freecodecamp](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/) about naming conventions. It will save you a lot of time in the future.

Reply:
Great job. This comment is constructive and delivered in a great manner. It explains why the suggested change is required, which will benefit the developer.